neo/SmartContract/NefFile.cs

```diff
@@ -0,0 +1,98 @@
+﻿using Neo.IO;
+using System;
+using System.IO;
+
+namespace Neo.SmartContract
+{
+    public class NefFile : ISerializable
+    {
+        public enum NefMagic : int
+        {
+            /// <summary>
+            /// NEO Executable Format 3
+            /// </summary>
+            NEF3 = 0x4E454633
```

Comment:
```suggestion
            NEF3 = 0x3346454E
```
This is supposed to be little endian. Needs fixing.

Reply:
I think that is ascii header

Reply:
Even webassembly does the same, its header is also little-endian, so as the platform.